plugins/node/instrumentation-mongoose/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@opentelemetry/instrumentation-mongoose",
   "version": "0.38.1",
-  "description": "OpenTelemetry automatic instrumentation package for mongoose",
+  "description": "OpenTelemetry instrumentation for `mongoose` database client for MongoDB",
```

Comment:
Since this package emits `db` namespaced spans,  tried to include the word `database` in the description. perhaps we can write 

```
OpenTelemetry instrumentation for `mongoose` database object data modeling (ODM) library for MongoDB
```

to keep it consistent with other database instrumentations?

Reply:
changed to "OpenTelemetry instrumentation for `mongoose` database object data modeling (ODM) library for MongoDB"